docs/guide/http_cache.md

```diff
@@ -687,6 +687,12 @@ set req.http.X-User-Hash = "b1731d46b0e7a375a5b024e950fdb8d49dd25af85a5c7dd5116a
 
 **5.** Restart the Varnish server and everything should work fine.
 
+##### Known limitations of the user hash generation
+
+A few limitations are known:
+
+* If you are using uri based site accesses matching, repository must be shared across one domain, because `/_fos_user_context_hash` is non siteaccess aware by default (see `ezpublish.default_router.non_siteaccess_aware_routes` parameter).
```

Comment:
Almost, but rule can be made more correct with:
`If you are using uri based site accesses matching, default siteaccess on the domain needs to point to same repository, because `

Reply:
Can maybe also add info in some form that Varnish does not have knowledge about siteaccesses, or that it might be part of uri, so this is the reason it has to be this way for this feature.